nethttp/client.go

```diff
@@ -121,7 +121,10 @@ func (t *Transport) RoundTrip(req *http.Request) (*http.Response, error) {
 	}
 	tracer, ok := req.Context().Value(keyTracer).(*Tracer)
 	if !ok {
-		return rt.RoundTrip(req)
+		req, tracer = TraceRequest(
+			opentracing.GlobalTracer(),
+			req,
+		)
 	}
 
 	tracer.start(req)
```

Comment:
This isn't `tracer.sp.Finished()` in a single case, that is on no error and non-HEAD request.